docs/create_rules.md

```diff
@@ -90,3 +107,42 @@ def sql_has_reasonable_number_of_lines(model: Model, max_lines: int = 200) -> Ru
             message=f"SQL query too long: {count_lines} lines (> {max_lines})."
         )
 ```
+
+### Filtering models
+
+Custom and standard rules can be configured to have model filters. Filters
+allows setting models of the project to be ignored by a given rule.
+
+Filters are created using the same discovery mechanism and interface as custom
+rules, except they do not accept parameters. Similar to Python's built-in
+`filter` function, when the filter evaluation returns `True` the model should be
+evaluated, otherwise it should be skipped.
+
+```python
+from dbt_score import ModelFilter, model_filter
+
+@model_filter
+def only_schema_x(model: Model) -> bool:
+    """Only applies a rule to schema X."""
+    return model.schema.lower() == 'x'
+
+class SkipSchemaY(ModelFilter):
+    description = "Applies a rule to every schema but Y."
+    def evaluate(self, model: Model) -> bool:
+      return model.schema.lower() != 'y'
```

Comment:
Do we know use-cases where the class version (as opposed to the decorator version) would be necessary?

Reply:
Not yet, but no harm in having it as it does not really change implementation and it follows the same design of `@rule`.